src/mpl/src/ParquetFP/src/DB.cxx

```diff
@@ -1895,7 +1896,8 @@ float DB::evalSteiner(bool useWts, bool scaleTerms)
         flute_x[i] = static_cast<double>(pointsOnNet[i].x);
         flute_y[i] = static_cast<double>(pointsOnNet[i].y);
       }
-      Tree flutetree = flute(pointsOnNet.size(), flute_x, flute_y, ACCURACY);
+      Tree flutetree
+          = flute.flute(pointsOnNet.size(), flute_x, flute_y, ACCURACY);
```

Comment:
Are you initializing this flute object? I believe you need to call the flute() function from an SteinerTreeBuilder object.

Reply:
As we discussed, yes, the flute object is not initialized this way, but this code was inside a conditional inclusion that's not used anymore (if USEFLUTE is defined).

So I removed all conditional includes that have this condition